kolibri/plugins/learn/assets/src/views/MasteredSnackbars/index.vue

```diff
@@ -32,7 +32,7 @@
           type="warning"
           class="alert"
         >
-          {{ $tr('signIn') }}
+          {{ coreCommon$tr('signInLabel') }}
```

Comment:
This change looks problematic, and we should be careful if it's indicative of other similar changes.

Here, the string `signIn` defined in this component was actually "Sign in or create an account to save points you earn", not "Sign in".

It's also still defined below. If this was a case where we were consolidating, it should have been deleted.

Reply:
I actually grep-replaced this by mistake. I changed another component's `$tr('signIn')` to the coreCommon on one or two other components and didn't realize that this was in the results.

Pushed a fix

Reply:
cool! should we delete `signInLabel: 'Sign in',`?

Reply:
@indirectlylit `signInLabel` in Core is used elsewhere